storage/gcloud/aio/storage/storage.py

```diff
@@ -172,6 +173,22 @@ async def _headers(self) -> Dict[str, str]:
             'Authorization': f'Bearer {token}',
         }
 
+    async def get_buckets(
```

Comment:
Looks like you're calling the "list bucket" operation here. `list_buckets()` might be a more accurate name.

Reply:
The thing with that name is that I'm not returning a `List[]` object. That's why I changed for `get_buckets()`. Are we ok with no returning a `List[]`?

Reply:
Yup, that's fine IMO, though I welcome anyone else's thoughts! You are still calling "the bucket listing API method" in this function, regardless of the exact data type you're returning it as.

Reply:
Good reasoning. It convinced me. I will return a `List[]` object which is the one retuned by the google API.